parquet/src/test/java/org/apache/iceberg/parquet/TestBloomRowGroupFilter.java

```diff
@@ -109,7 +109,7 @@ public class TestBloomRowGroupFilter {
           optional(22, "timestamp", Types.TimestampType.withoutZone()),
           optional(23, "timestamptz", Types.TimestampType.withZone()),
           optional(24, "binary", Types.BinaryType.get()),
-          optional(25, "int_decimal", Types.DecimalType.of(8, 2)),
+          optional(25, "int-decimal", Types.DecimalType.of(8, 2)),
```

Comment:
Minor quibble here, let's not re-use this field and instead add a new field like we did with "non_bloom", "struct_not_null" etc ...
So something like
```java
optional(28, "incompatible-name", StringType.get())
```

Reply:
Changed. Thanks